src/main/java/edu/kis/vh/nursery/IStack.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.vh.nursery;
+
+public interface IStack {
```

Comment:
Names of operations do not correspond to operating the stack, thus do not reflect the interface name.